XamlControlsGallery/DataModel/ControlInfoData.json

```
@@ -1694,7 +1694,7 @@
             },
             {
               "Title": "Guidelines",
-              "Uri": "https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/active/WebView2/WebView2_spec.md"
+              "Uri": "https://docs.microsoft.com/en-us/microsoft-edge/webview2/gettingstarted/winui"
```

We try to avoid using localized URLs as this interferes with the website loading the link in the language chosen by the user if they are signed in.

```suggestion
              "Uri": "https://docs.microsoft.com/microsoft-edge/webview2/gettingstarted/winui"
```